src/packages/core/utils/string/generate-umbraco-alias.function.ts

```diff
@@ -1,18 +0,0 @@
-import { toCamelCase } from './to-camel-case/to-camel-case.function.js';
-
-/**
- *
- * @param text
- */
-export function generateAlias(text: string) {
-	//replace all spaces characters with a dash and remove all non-alphanumeric characters, except underscore. Allow a maximum of 1 dashes or underscores in a row.
-	//Do we really want that and not the original way of doing things (camelCase)?
-
-	return toCamelCase(text);
-
-	return text
```

Comment:
Dont we miss this? Or just some of it?

Reply:
I think the end result looks ok. 

<img width="797" alt="Screenshot 2024-10-04 at 09 14 41" src="https://github.com/user-attachments/assets/5b951062-de85-4334-8533-b8e53b75922e">